requests/utils.py

```diff
@@ -20,10 +20,11 @@
 
 from . import __version__
 from . import certs
+# noinspection PyUnresolvedReferences
```

Comment:
On the other hand, we *should* put a comment above this import that says that we know that this is unused in this file, but we want it here for backward compat purposes.